src/components/Filters/Filters.js

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+import {connect} from 'react-redux'
+import FilterStyle from './FiltersStyle'
+import Button from '../../styledComponents/Button'
+import {setProductsSort} from './actionCreators'
+
+const mapStateTopProps = (state) => ({
+  sortType:state.sortProducts.sortProductsType
+})
+
+const mapDispatchToProps = (dispatch) => {
+  return {
+    setProductsSort: (sortType) => dispatch(setProductsSort(sortType))
+  }
+}
+const Filters = (props) => {
+    return (
+      <FilterStyle>
+        <span>Sort by price:</span>
+
+        <Button active={props.sortType === 'PRICE_ASC' ? 'active' : false}
```

Comment:
I think capitalize is enough to see the difference (especially if it's general React rule) and complexity can be confusing if it's still just button. For example, react-bootstrap is using the same naming.